rfcs/react-components/components/dev-docs.md

```diff
@@ -0,0 +1,69 @@
+As a new developer to the Fluent repo, but long time consumer and fan, the supporting documentation has room for opportunity. To be productive, I've had to randomize other members of the core team which slows us both down.
+
+This RFC seeks to lay out a plan to update the Fluent dev docs. The ultimate goal is to streamline contributors path to success.
+
+## Goals of the new system:
+
+- Be more accessible to developers in their work flow.
+- Help developers onboard faster without distracting core team members.
+- Provide technical documentation and guides for common patterns and practices that do not make sense to live in examples or code samples.
+- Focus attention on areas under active development (web and react components)
+
+## Non-Goals
+
+- Provide an ongoing communications platform for project statuses, road maps and planning.
+- Provide documentation relevant to other disciplines outside engineering.
+- Provide documentation around consuming the library. These should live externally in the Storybook.
+- Provide documentation around filing bugs or how to engage with the team.
+
+## Current issues:
+
+- Mixture of v6, 7, 8 and 9 specific docs.
+- Docs seemed to have stopped being updated in the early days of v9 development.
+- Mixture of process, set up, and reference docs.
+- RFC’s seem to be a large part of the Fluent process now, but there’s no clear way for an outsider to observe or contribute to them.
+- Use of Github’s wiki isn't the best technical solution.
+
+## Proposed changes
+
+### Place all dev documents in markdown files in a `__docs__` folder at the root of the repo.
```

Comment:
Let's also follow the microsoft OSS pattern and use a `CONTRIBUTING.md` in the root of our repo. We can use relative links in the markdown file to whatever folder we choose for other documentation files

Reply:
Yep, I like this a lot - we'll hold off on adding it until all the docs we want it to point to are in place.